src/components/booksform.js

```diff
@@ -1,32 +1,87 @@
 import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
this component should be in `src/containers` folder (this was to be corrected in milestone 2). Please it must have been forgotten. Please relocate it....thanks

Reply:
you'll also have to fix any import links subsequently

Reply:
Ok got it.

Reply:
This issue has been sorted.